### PR TITLE
RAC-5104 -  Converted CIT formatted apiv2 scripts to FIT format -tags, schema, swagger scripts 

### PR DESCRIPTION
**Pull request by hohene**
Converted the workflows CIT test scripts to FIT format.
This is first part of committing these changed scripts to show the diffs.
tags, schema, swagger CIT test scripts.

### CODE REVIEW

**Comment by hohene** [comment]
@larry-dean @keedya @jimturnquist @johren @derrickostertag

File: test/tests/api/fit-staging/v2_0/schema_tests.py [hunk 1]
```diff
@@ -1,58 +1,65 @@
...
+from nose.plugins.attrib import attr
+from nosedep import depends
+
+logs = flogging.get_loggers()
 
-LOG = Log(__name__)
 
-@test(groups=['schemas_api2.tests'])
-class SchemaTests(object):
+# @test(groups=['schemas_api2.tests'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/schema_tests.py]: remove (per jeanne)

File: test/tests/api/fit-staging/v2_0/schema_tests.py [hunk 2]
```diff
@@ -1,58 +1,65 @@
...
+    @classmethod
+    def setUpClass(cls):
+        cls.__client = config.api_client
+        cls.__schemaList = None
 
     def __get_data(self):
         return loads(self.__client.last_response.data)
 
-    @test(groups=['2.0.list_schemas'])
+    # @test(groups=['2.0.list_schemas'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/schema_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/schema_tests.py [hunk 3]
```diff
@@ -1,58 +1,65 @@
...
         schemas = self.__get_data()
-        LOG.debug(schemas,json=True)
-        assert_not_equal(0, len(schemas), message='Schema list was empty')
-        self.__schemaList = schemas
+        logs.debug(dumps(schemas, indent=4))
+        self.assertNotEqual(0, len(schemas), msg='Schema list was empty')
+        self.__class__.__schemaList = schemas
 
-    @test(groups=['2.0.get_schema'], depends_on_groups=['2.0.list_schemas'])
+    # @test(groups=['2.0.get_schema'], depends_on_groups=['2.0.list_schemas'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/schema_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/schema_tests.py [hunk 4]
```diff
@@ -1,58 +1,65 @@
...
-            LOG.debug(schema_ref,json=True)
-            id = schema_ref.get('title')
-            assert_true('title' in schema_ref.keys(), message='title not found in schema')
-            assert_true('definitions' in schema_ref.keys(), message='definitions not found in schema')
+            logs.debug(dumps(schema_ref, indent=4))
+            self.assertIn('title', schema_ref.keys(), msg='title not found in schema')
+            self.assertIn('definitions', schema_ref.keys(), msg='definitions not found in schema')
 
-    @test(groups=['2.0.get_schema_invalid'], depends_on_groups=['2.0.list_schemas'])
+    # @test(groups=['2.0.get_schema_invalid'], depends_on_groups=['2.0.list_schemas'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/schema_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/swagger_tests.py [hunk 5]
```diff
@@ -1,30 +1,40 @@
...
-        print config.api_root
-        print config.host
+
+from config.api2_0_config import config
+from nose.plugins.attrib import attr
+
+logs = flogging.get_loggers()
+
+
+# @test(groups=['swagger.tests'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/swagger_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/tags_tests.py [hunk 6]
```diff
@@ -1,118 +1,146 @@
...
+from nosedep import depends
+
+logs = flogging.get_loggers()
 
-LOG = Log(__name__)
 
-@test(groups=['tags_api2.tests'])
-class TagsTests(object):
-    def __init__(self):
+# @test(groups=['tags_api2.tests'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/tags_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/tags_tests.py [hunk 7]
```diff
@@ -1,118 +1,146 @@
...
+                logs.debug(dumps(tagsWithRules, indent=4))
                 Api().create_tag(body=tagsWithRules)
                 tag_data = self.__get_data()
-                assert_equal(n.get('id'), tag_data["name"], "Failed creating tag")
-        assert_raises(rest.ApiException, Api().create_tag, body='fooey')
+                self.assertEqual(node.get('id'), tag_data["name"], "Failed creating tag")
+        self.assertRaises(rest.ApiException, Api().create_tag, body='fooey')
 
-    @test(groups=['test-tags-api2'], depends_on_groups=['create-tag-api2'])
+    # @test(groups=['test-tags-api2'], depends_on_groups=['create-tag-api2'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/tags_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/tags_tests.py [hunk 8]
```diff
@@ -1,118 +1,146 @@
...
-        assert_equal(200, rsp.status, message=rsp.reason)
+        self.assertEqual(200, rsp.status, msg=rsp.reason)
         for i in xrange(len(updated_tags)):
-            assert_equal(updated_tags[i]['rules'][0]['path'], 'dmi.System Information.Manufacturer', message='Could not find the tag')
+            self.assertEqual(updated_tags[i]['rules'][0]['path'], 'dmi.System Information.Manufacturer',
+                             msg='Tag {} has incorrect path: {}'.format(updated_tags[i],
+                                                                        updated_tags[i]['rules'][0]['path']))
 
-    @test(groups=['test-tags-tagname-api2'], depends_on_groups=['test-tags-api2'])
+    # @test(groups=['test-tags-tagname-api2'], depends_on_groups=['test-tags-api2'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/tags_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/tags_tests.py [hunk 9]
```diff
@@ -1,118 +1,146 @@
...
                 rsp = self.__client.last_response
                 tag = self.__get_data()
-                assert_equal(200, rsp.status, message=rsp.reason)
-                assert_equal(tag.get('name'), n.get('id'), message='Could not find the tag')
+                self.assertEqual(200, rsp.status, msg=rsp.reason)
+                self.assertEqual(tag.get('name'), node.get('id'),
+                                 msg='Could not find the tag: {}'.format(node.get('id')))
 
-    @test(groups=['test-nodes-tagname-api2'], depends_on_groups=['test-tags-tagname-api2'])
+    # @test(groups=['test-nodes-tagname-api2'], depends_on_groups=['test-tags-tagname-api2'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/tags_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/tags_tests.py [hunk 10]
```diff
@@ -1,118 +1,146 @@
...
                     nodesList = self.__get_data()
                     tagsList = nodesList[0]['tags']
-                    checkTag = n.get('id') in tagsList
-                    assert_equal(True, checkTag, message=rsp.reason)
+                    self.assertIn(tagged_node.get('id'), tagsList,
+                                  msg="Tag {} not in taglist for node {}".format(node.get('id'),
+                                                                                 tagged_node.get('id')))
 
-    @test(groups=['test_tags_delete'], depends_on_groups=['test-nodes-tagname-api2'])
+    # @test(groups=['test_tags_delete'], depends_on_groups=['test-nodes-tagname-api2'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/tags_tests.py]: remove

**Pull request merged**